### ISSUE TITLE
Note Length Discoveries

### ISSUE THREAD

**Issue by nobodycarestrue** [hamstah/Orca#8]
in the io.midi.js file:

function convertLength (val, bpm) {
    // TODO get bpm from daw midi
    if (! bpm) {
      bpm = 120;
    }
    return (60000 / 120) * (val / 15)

where it says  "60000 / x", x should be 1/4th of bpm.
So for the example of bpm = 120, x = 30.

This is how to get a note length value of 'f' to equal ~1 bar instead of the current quarter note maximum.

_{There's also some kind of issue with an 'f' value sometimes overlapping with the next note creating a super short note length on that note at random intervals.}_

**Comment by sense-amr** [comment]
would be AWESOME if the DAW could parse this through .. to orca and then Orca adjusts this value dynamically :) doable ? @hamstah ?

**Comment by nobodycarestrue** [comment]
Maybe disregard this whole issue. A bunch of timing issues start happening with increased note length. :-(

**Comment by sense-amr** [comment]
yeah trial and error testing and seeing man .. its all part of it .. nice first issue :) good work
